resources/web/docs_js/index.js

```diff
@@ -299,6 +337,25 @@ $(function() {
     $('a.edit_me_private').show();
   }
 
+  // scroll to selected TOC element; if it doesn't exist yet, wait and try again
+  // window.width must match the breakpoint of `.sticky-top-md`
+  if($(window).width() >= 769){
+    var scrollToSelectedTOC = setInterval(() => {
+      if ($('.current_page').length) {
+          // Get scrollable element
+          var container = document.querySelector("#left_col");
+          // Get active table of contents element
+          var activeItem = document.querySelector(".current_page")
+          // If the top of the active item is out of view
+          if (container.offsetHeight - 100 <= activeItem.offsetTop) {
```

Comment:
Where does the `100` come from?

Reply:
The value `100` is somewhat arbitrary. It specifies that if an active item is in the bottom 100px of the visible portion of the table of contents, we should scroll to avoid the active item being _barely_ visible like this (where the active item, "Migration guide" is technically within view but barely):

![image](https://user-images.githubusercontent.com/10479155/183947742-ed841cbd-3b57-4147-828b-6bd4e206310a.png)